### PR TITLE
Add general document structure

### LINKED ISSUE #1: Bootstrap the site layout structure
- Create all basic HTML and CSS files 
- Add global containers, etc: header, footer, main, sidebar
- Include CSS Reset or Normalize

### PR DESCRIPTION
**Pull request by Berezhnyi**
Resolves #1

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8" />
+	<!--[if lt IE 9]><script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.min.js"></script><![endif]-->
+	<title></title>
+	<meta name="keywords" content="" />
+	<meta name="description" content="" />
+	<link href="style.css" rel="stylesheet">
```
Comment (constgen) [inline src/index.html]: If you have created `index.js` then include it here

File: src/styles/style.css [hunk 2]
```diff
@@ -0,0 +1,91 @@
+/* Eric Meyer's CSS Reset */
+html, body, div, span, applet, object, iframe,
```
Comment (constgen) [inline src/styles/style.css]: You don't need this Reset when you have your own `reset.css`

File: src/index.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8" />
+	<!--[if lt IE 9]><script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.min.js"></script><![endif]-->
+	<title></title>
+	<meta name="keywords" content="" />
+	<meta name="description" content="" />
+	<link rel="stylesheet" href="./styles/reset.css">
+	<link rel="stylesheet" href="./styles/style.css">
```
Comment (constgen) [inline src/index.html:10]: If you have created `index.js` then include it here

**Review by constgen — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 3 above)
Comment (Berezhnyi) [reply src/index.html:10]: Added <script>

File: src/styles/style.css (hunk 2 above)
Comment (Berezhnyi) [reply src/styles/style.css]: Removed Eric Meyer's CSS Reset

**Review by constgen — APPROVED** [review]

**Pull request merged**